loglevels-logback/src/main/scala/akka/management/loglevels/logback/LogLevelRoutes.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2017-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.management.loglevels.logback
+
+import akka.actor.ExtendedActorSystem
+import akka.actor.Extension
+import akka.actor.ExtensionId
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Route
+import akka.management.scaladsl.ManagementRouteProvider
+import akka.management.scaladsl.ManagementRouteProviderSettings
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.unmarshalling.Unmarshaller
+import ch.qos.logback.classic.Level
+import ch.qos.logback.classic.LoggerContext
+import org.slf4j.LoggerFactory
+
+object LogLevelRoutes extends ExtensionId[LogLevelRoutes] {
+  override def createExtension(system: ExtendedActorSystem): LogLevelRoutes =
+    new LogLevelRoutes
+
+  private implicit val levelFromStringUnmarshaller: Unmarshaller[String, Level] =
+    Unmarshaller.strict { string =>
+      Level.valueOf(string)
```

Comment:
maybe validate that this is valid? Otherwise it will silently set it to DEBUG if someone typos ERROR which may blow up their logging infra

Reply:
Ah you are right, I expected it to throw for invalid value for some reason.